### DISCUSSION TITLE
Smart Connections: Suggestions only within a subordinate folder hierarchy / Each folder hierarchy has its own smart connections system

### DISCUSSION THREAD

**Discussion by BRogers80** [brianpetro/obsidian-smart-connections#535] [Ideas]
Dear team,

I discovered this plugin a few days ago and really like it. I am often too lazy to tag and interlink my notes, so this is a game changer for me.
I use obsidean for all my meeting notes, strategy meetings and so on for my customers. Therefor I have a folder structure which represents every customer in the first top level folder, followed by folders with meeting minutes, stratetegy and so on.

- customer A
-- protocols
-- strategy

- customer B
-- protocols
-- strategy

- customer C
-- protocols
-- strategy

I would really like to get smart connections to just suggest me links and contents to one customer if the note is in any of the subfolders. So if I have a protocol in customer C, the system should just suggest documents in the customer C folder and all subfolders and files of customer C.
I also can image to filter based on tags, if its possible.

Thank you in advance for your great work

Florian

**Comment by brianpetro** [comment]
Hey @BRogers80 

I'm happy to hear that you're enjoying Smart Connections 😊

It seems like you're looking for the ability to limit the Smart View to a folder or tags.

This is something that is in the works.

In the Smart Chat, you can say something like "Based on my notes in /folder/ tell me..." and the chat will limit its context lookup to that folder.

Similarly, in the Smart View, I plan on adding "filters" that would allow you to specify a folder to limit the output based on that folder.

🌴

**Comment by BRogers80** [comment]
Thank you for your response.

That sounds great. It would be nice if the filter of the smart view can also include all subfolders.

Thank you for the great work
